applications/Omniverse DLT.md

```diff
@@ -0,0 +1,264 @@
+
+### Team Code Repos
+
+To be more clearly, we have created a new organization to manage the related code:  
+- https://github.com/Omniverse-Web3-Labs/  
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- https://github.com/xiyu1984
+- https://github.com/virgil2019
+- https://github.com/kay404
+
+## Development Status :open_book:
+
+Before apply for this grant, we have done some neccessary researches and built some prototypes to verify them.  
+  - [Simulation of Omniverse DLT on Substrate Pallets](https://github.com/Omniverse-Web3-Labs/omniverse-swap)
+  - [Off-Chain Simulation of O-AMM mathematic model](https://github.com/Omniverse-Web3-Labs/o-amm)
+  - [On-Chain Simulation of O-AMM mathematic model](https://github.com/Omniverse-Web3-Labs/O-AMM-ParaSim)
+
+After the simulation, we believe that the Omniverse DLT is achievable on Polkadot and other chains.  
+
+## Development Roadmap :nut_and_bolt:
+### Overview
+
+- **Total Estimated Duration:** 4 months
+- **Full-Time Equivalent (FTE):**  3 FTE
+- **Total Costs:** 30,000 USD. 
+
+### Milestone 1 — O-DLT: Substrate Pallet and Solidity smart contract
+
+- **Estimated duration:** 2 month
+- **FTE:**  3
+- **Costs:** 15,000 USD
+
+> :exclamation: **The default deliverables 0a-0d below are mandatory for all milestones**, and deliverable 0e at least for the last one. 
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | GPLv3 |
+| **0b.** | Documentation | - We will provide docs to explain how O-DLT works in a high-level <br> - We will provide dev docs to explain how to develop an O-DLT component in Substrate Pallet/solidity smart contract <br> - We will provide tutorial docs to explain how to use it |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0d. | Article | We will publish an **article** that explains what was done/achieved in this milestone as part of the grant. |
+| 1. | Substrate module: omniverse assets | We will create a Substrate module that will manage the Omniverse tokens, which will be derived from standard `pallet assets` |
+| 2. | Substrate module: omniverse protocol | We will create a Substrate module that will manage the underlying omniverse functions like omniverse account, omniverse signature, omniverse verification, etc. |
+| 3. | Substrate chain | Modules omniverse assets, protocol of our custom chain will interact in such a way that can make creation, omniverse mint, burn, and transferring for o-tokens |
+| 4. | Solidity Smart Contracts for Moonbeam | We will create the fully O-DLT protocol in solidity tech stack for Moonbeam. It will have the same functions as in `Substrate Pallets` |
```

Comment:
> One thing to keep in mind is that we don't usually support the implementation of Solidity-based contracts, since we're not really experts in that field. One potential source of funding for the Solidity-based part of your proposal would be the [Moonbeam grants program](https://moonbeam.foundation/grants/).

I will remove the solidity part.